packages/react-ui/src/app/features/builder/blocks-selector/index.tsx

```diff
@@ -245,17 +245,22 @@ const BlockSelector = ({
       metadata?.filter((stepMetadata) => {
         if (selectedTag === ALL_KEY) return true;
 
+        const includesTag = (
+          stepMetadata as BlockStepMetadata
+        ).categories?.includes(selectedTag);
+
         if (selectedTag === BlockCategory.WORKFLOW) {
-          return [
-            ActionType.LOOP_ON_ITEMS,
-            ActionType.SPLIT,
-            ActionType.BRANCH,
-          ].includes(stepMetadata.type as ActionType);
+          return (
```

Comment:
@rSnapkoOpenOps please review, it's for this:
![image](https://github.com/user-attachments/assets/3897a0f5-dcb6-4040-80d8-86e5bd5bb3c5)